storage/engine/cgo_flags.go.in

```diff
@@ -0,0 +1,10 @@
+package engine
+
+// This file is generated because in go 1.4, there is no way to
+// include a relative path in cgo's LDFLAGS. (go 1.5 will include
+// variable expansion in these flags).
+
+// #cgo CXXFLAGS: -std=c++11
+// #cgo CPPFLAGS: -I ../../_vendor/usr/include
```

Comment:
The compilation phase of cgo is run on a per-package basis, with that package as the working directory. Linking, however, is done in the directory of the final build target (either a test or the top-level `cockroach` binary). Whatever we put in LDFLAGS is copied verbatim so it must work no matter what the current directory is.